ivy/functional/frontends/torch/func_wrapper.py

```diff
@@ -243,6 +243,40 @@ def outputs_to_native_arrays_torch(*args, **kwargs):
     return outputs_to_native_arrays_torch
 
 
+def to_ivy_shape(fn: Callable) -> Callable:
+    """
+    Wrap `fn` so it receives `ivy.Shape` instead of Native shapes.
+
+    Wrap `fn` so that any `torch.Size` arguments are converted to
+    `ivy.Shape` instances.
```

Comment:
To be clear, please refer to the shapes we are converting as `torch_frontend.Size`, not `torch.Size` (this refers to the original torch class) neither NativeShape (this refers mostly to the backends).
Also the first sentence is not necessary at all.